sceptre/scipool/config/develop/sc-tag-options.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   - "develop/sc-portfolio-scheduled-jobs.yaml"
 sceptre_user_data:
   CostCenters: !from_json
-    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true'
+    - !request 'https://mips-api.finops.sageit.org/tags?limit=25&enable_other_code=true&priority_codes=122500'
```

Comment:
could you please add a comment explaining why we are doing this?  maybe just a link to the issue might be good enough

Reply:
done